conf/routes

```diff
@@ -32,7 +32,10 @@ GET  /contribute/recurring/thankyou                 controllers.Application.reac
 GET  /contribute/recurring/existing                 controllers.Application.reactTemplate(title="Support the Guardian | Existing Contributor", id="regular-contributions-existing-page", js="regularContributionsExistingPage.js")
 POST /contribute/recurring/create                   controllers.RegularContributions.create
 GET  /contribute/recurring/status                   controllers.RegularContributions.status(jobId: String)
-GET  /contribute/recurring/pending                 controllers.Application.reactTemplate(title="Support the Guardian | Thank You", id="regular-contributions-pending-page", js="regularContributionsPendingPage.js")
+GET  /contribute/recurring/pending                  controllers.Application.reactTemplate(title="Support the Guardian | Thank You", id="regular-contributions-pending-page", js="regularContributionsPendingPage.js")
+GET  /contribute/send-marketing                     controllers.IdentityController.submitMarketing(email: String)
```

Comment:
I'm concerned that this endpoint is completely opening up the identity api consent emailer endpoint, without any security

Reply:
Also, should this not be a POST?